Library/Formula/sipsak.rb

```diff
@@ -1,8 +1,8 @@
 require 'formula'
 
 class Sipsak < Formula
-  homepage 'http://sipsak.org/'
```

Comment:
The sipsak.org host is not longer available; download appear now to be hosted on SF.